addon/test-waiter.ts

```diff
@@ -128,4 +132,10 @@ export default class TestWaiter<T = Token> implements ITestWaiter<T> {
   reset(): void {
     this.items.clear();
   }
+
+  private getCompletedOperations(token: T) {
```

Comment:
Let's underscore this method (the TS `private` only helps us with TS consumers).

Reply:
Sure. Can do.